content/articles/pse-june-2025-newsletter.md

```diff
@@ -0,0 +1,90 @@
+---
+authors: ["PSE Comms Team"]
+title: "PSE June 2025 newsletter"
+image: "/articles/pse-june-2025/cover.webp"
+tldr: "Check out what PSE teams have been focused on in June 2025!"
+date: "2025-06-13"
+tags:
+  [
+    "newsletter",
+    "post quantum cryptography",
+    "private proof delegation",
+    "client side proving",
+    "machina io",
+    "voprf",
+    "tlsnotary",
+    "maci",
+    "zk-kit",
+    "mpc framework",
+    "mopro",
+    "semaphore",
+  ]
+projects:
+  [
+    "post-quantum-cryptography",
+    "private-proof-delegation",
+    "client-side-proving",
+    "machina-io",
+    "voprf",
+    "tlsn",
+    "maci",
+    "zk-kit",
+    "mpc-framework",
+    "mopro",
+    "semaphore",
+  ]
+---
+
+Here’s a round-up of what PSE teams have been up to in June!
+
+## 🧪 Research Highlights
+
+### [Client-Side Proving](https://pse.dev/en/projects/client-side-proving)
+
+Our Client-Side Proving team [published benchmarks](https://pse.dev/blog/efficient-client-side-proving-for-zkid) for post-quantum zk-SHA256 circuits, helping assess feasibility for mobile and other tighter bandwidth setups. In our next phase, we will extend our benchmarks beyond zkID and explore client-side proving for Ethereum.
+
+### [Verifiable OPRF (vOPRF)](https://pse.dev/en/projects/voprf)
+
+A [new fork](https://github.com/privacy-scaling-explorations/stealthnote) of [Stealthnote](https://stealthnote.xyz/) is underway. StealthNote is an app that allows people in an organization to anonymously broadcast messages using zero-knowledge proofs.
```

Comment:
@kichong we should add context on the goal here (to add vOPRF to Stealthnote - that's why we're forking it) 😃